### PR TITLE
Editing Pandas Documentation homepage

### PR DESCRIPTION
**Pull request by MOHED1224**
Trying to improve the user experience and interaction with the webpage.

1.Edited button hover effect.
2. Make links open in new tabs instead of redirect in the current webpage because this may be not good for some people.
3. Added data manipulation note in the introduction section.

### CODE REVIEW

**Comment by MOHED1224** [comment]
Just removing the errors in the first 2 commits.

**Comment by jreback** [comment]
so this is just a change to hover (1) right?

**Comment by jreback** [comment]
cc @datapythonista

**Comment by MOHED1224** [comment]
> so this is just a change to hover (1) right?
yes

**Comment by datapythonista** [comment]
Thanks @MOHED1224. I think it's fine here, but in general we prefer to not mix PRs with unrelated changes, even if they are small.

@jorisvandenbossche is this hover effect something that should be implemented here, or in the original sphinx theme?

**Comment by jorisvandenbossche** [comment]
Sorry for the slow reply here

>  is this hover effect something that should be implemented here, or in the original sphinx theme?

At the moment the "custom-button" (and its css) is defined here in pandas, so a hover effect on it is also to be implemented in pandas. 
Although long term we should probably look into replacing it with the buttons provided by sphinx-panels: https://sphinx-panels.readthedocs.io/en/latest/#link-buttons

File: doc/source/index.rst.template [hunk 1]
```diff
@@ -20,7 +20,7 @@ pandas documentation
 `Mailing List <https://groups.google.com/g/pydata>`__
 
 :mod:`pandas` is an open source, BSD-licensed library providing high-performance,
-easy-to-use data structures and data analysis tools for the `Python <https://www.python.org/>`__
+easy-to-use data structures, data manipulation and data analysis tools for the `Python <https://www.python.org/>`__
```
Comment (jorisvandenbossche) [inline doc/source/index.rst.template:23]:
```suggestion
easy-to-use data structures, data manipulation and analysis tools for the `Python <https://www.python.org/>`__
```

**Comment by datapythonista** [comment]
I'm going to close this. @MOHED1224 let's do this properly even if it's small. Can you open an issue with screenshots of the button you think should be improved, with and without your proposed hover. And tag me there, if there is agreement please reopen the PR only with those changes, and nothing else.

**Pull request closed without merging**